core/io/file_access_encrypted.cpp

```diff
@@ -310,6 +300,6 @@ Error FileAccessEncrypted::_set_unix_permissions(const String &p_file, uint32_t
 
 FileAccessEncrypted::~FileAccessEncrypted() {
 	if (file.is_valid()) {
```

Comment:
Can be removed.

Reply:
No, `_release()` do not have a check only `release()`, but I guess there's no need to keep both.

Reply:
Ah right. Maybe re-add `_close` for consistency with other classes and remove `_release`?